react/components/ReviewBlock.tsx

```diff
@@ -500,12 +501,20 @@ const ReviewBlock: FunctionComponent<WrappedComponentProps & any> = ({
     }
 
     if (columnsToBeHidden.indexOf('totalQuantity') === -1) {
+      const totalQuantity = intl.formatMessage({
+        id: 'store/quickorder.review.label.totalQuantity',
+      }) as string
+
+      const [quantity, total] = totalQuantity.split(' ')
+
       tableSchema.properties.totalQuantity = {
         type: 'float',
-        title: intl.formatMessage({
-          id: 'store/quickorder.review.label.totalQuantity',
-        }),
-        width: 100,
+        title: (
+          <>
+            {quantity} <span style={{ display: 'block' }}>{total}</span>
```

Comment:
A intenção aqui  é só garantir que ele quebre em duas linhas mesmo?

Reply:
Pq se for, não precisa disso tudo né? O split e tal, uma alternativa pode ser só colocar esse cara em uma div e colocar um `whiteSpaces: `break-space``

Alguma coisa assim:

```
 title: (
          <div style={{ whiteSpace: 'break-spaces' }}>
            {intl.formatMessage({
              id: 'store/quickorder.review.label.totalQuantity',
            })}
          </div>
        ),
```

Reply:
resolvido